### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**
Updates for the 6.2.6-RC1 and 7.0.6-RC1 releases

### CODE REVIEW

File: download/index.md [hunk 1]
```diff
@@ -81,45 +82,51 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
...
+### Redis Stack 6.2.6-RC1
+
+* [macOS x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.catalina.x86_64.zip)
+* [macOS arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.monterey.arm64.zip)
+* [Ubuntu xenial x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.xenial.x86_64.tar.gz)
+* [Ubuntu bionic x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.bionic.x86_64.tar.gz)
+* [Ubuntu bionic arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.bionic.arm64.tar.gz)
+* [Ubuntu focal x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.focal.x86_64.tar.gz)
+* [Ubuntu focal arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.focal.arm64.tar.gz)
+* [Ubuntu Snap x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.x86_64.snap)
```
Comment (chayim) [inline download/index.md:94]: Missing arm64 snap link

File: download/index.md [hunk 2]
```diff
@@ -81,45 +82,51 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
...
+* [AppImage x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1-x86_64.AppImage)
+* [Redhat/CentOS 7 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.rhel7.x86_64.tar.gz)
+* [Redhat/CentOS 8 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.rhel8.x86_64.tar.gz)
+* [Debian bullseye x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.bullseye.x86_64.tar.gz)
+* [Ubuntu Jammy x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.jammy.x86_64.zip)
+* [Ubuntu Jammy arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.jammy.arm64.zip)
 * [redis-stack](https://hub.docker.com/r/redis/redis-stack) and [redis-stack-server](https://hub.docker.com/r/redis/redis-stack-server) x86_64 and arm64 dockers are available on [Dockerhub](https://hub.docker.com/u/redis)
```
Comment (chayim) [inline download/index.md:101]: This should probably say redis-stack docker?

File: download/index.md [hunk 3]
```diff
@@ -81,45 +82,51 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
...
+#### Download Redis Stack Server 7.0.6-rc1:
+
+* [macOS x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.catalina.x86_64.zip)
+* [macOS amd64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.monterey.arm64.zip)
+* [Ubuntu xenial x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.xenial.x86_64.tar.gz)
+* [Ubuntu bionic x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.bionic.x86_64.tar.gz)
+* [Ubuntu bionic arm64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.bionic.arm64.tar.gz)
+* [Ubuntu focal x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.focal.x86_64.tar.gz)
+* [Ubuntu focal arm64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.focal.arm64.tar.gz)
+* [Ubuntu Snap x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.x86_64.snap)
```
Comment (chayim) [inline download/index.md:123]: Missing arm64 snap link

File: download/index.md [hunk 4]
```diff
@@ -81,45 +82,51 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
...
+* [AppImage x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1-x86_64.AppImage)
+* [Redhat/CentOS 7 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.rhel7.x86_64.tar.gz)
+* [Redhat/CentOS 8 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.rhel8.x86_64.tar.gz)
+* [Debian bullseye x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.bullseye.x86_64.tar.gz)
+* [Ubuntu Jammy x86_64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.jammy.x86_64.zip)
+* [Ubuntu Jammy arm64](https://packages.redis.io/redis-stack/redis-stack-server-7.0.6-RC1.jammy.arm64.zip)
```
Comment (chayim) [inline download/index.md:129]: Same docker comment

File: download/index.md [hunk 5]
```diff
@@ -25,12 +25,13 @@ Redis 7.0 includes several new user-facing features, significant performance opt
 
 Download the latest Redis Stack Server binaries here, or install with [Docker](/docs/stack/get-started/install/docker), [Homebrew](/docs/stack/get-started/install/mac-os), or [on Linux](/docs/stack/get-started/install/linux).
 
-### Stable (6.2.4)
+### Stable (6.2.6)
 
-Redis Stack Server extends Redis with modern data models such as document, graph, time series. Redis Stack also includes RedisInsight, a visualization tool for Redis. Read the [latest release notes](https://github.com/redis-stack/redis-stack/releases/tag/v6.2.4-v1), or download the latest 6.2.4 binaries:
+Redis Stack Server extends Redis with modern data models such as document, graph, time series. Redis Stack also includes RedisInsight, a visualization tool for Redis. Read the [latest release notes](https://github.com/redis-stack/redis-stack/releases/tag/v6.2.6-RC1), or download the latest 6.2.6 binaries:
```
Comment (chayim) [inline download/index.md:30]: Why is this 6.2.6 RC1 instead of V?

File: download/index.md (hunk 1 above)
Comment (ViktarStarastsenka) [reply download/index.md:94]: added

File: download/index.md (hunk 3 above)
Comment (ViktarStarastsenka) [reply download/index.md:124]: added, thanks

File: download/index.md (hunk 2 above)
Comment (ViktarStarastsenka) [reply download/index.md:102]: I guess it should be clear for users, because we say redis-stack and redis-stack-server dockers are available on Dockerhub

**Review by Levyyyo — APPROVED** [review]

**Review by Levyyyo — DISMISSED** [review]

**Review by Levyyyo — APPROVED** [review]

**Pull request closed without merging**